lib/TeleIrc.js

```diff
@@ -395,7 +391,7 @@ class TeleIrc {
     }
     this.tgEventListener.addListener(
       'photo',
-      this.tgPhotoHandler.ReportPhoto.bind(this.tgPhotoHandler)
+      this.tgPhotoHandler.RelayPhotoMessage.bind(this.tgPhotoHandler)
```

Comment:
Ah, the dangers of large code reviews - missing simple things like this

Reply:
:+1: